docusaurus/docs-internals/editor-controls/multivalue-controls/padding.md

```diff
@@ -162,3 +218,144 @@ The `units` property is set to `"%"`, indicating that the configuration values w
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `padding` control is defined with several default values that specify its initial state. The `id` and `type` properties both indicate that this control deals with `padding` settings. The `default` object within the control contains the properties and their default values.
+
+```js
+{
+  id: "padding", 
+  type: "padding",
+  default: {
+    type: "ungrouped",
+    value: 10,
+    unit: "px",
+    top: 0,
+    topUnit: "px",
+    right: 0,
+    rightUnit: "px",
+    bottom: 0,
+    bottomUnit: "px",
+    left: 0,
+    leftUnit: "px"
```

Comment:
check one more time keys in `Padding/converters.ts` on `fromElementModel`

as example instead of `bottomUnit` should be `bottomSuffix`